web/js/validation.js

```diff
@@ -0,0 +1,4 @@
+(function () {
+    var regex = "[a-zA-Z0-9_]+";
+    $("#user_username").attr("pattern", regex);
```

Comment:
deze functie werkt niet ... de id van het veld voor de username in de route /vrijwilliger_worden is person_username.

Daarnaast lijkt het mij ook beter om - als je met een pattern werkt - gewoon het twig template aan te passen zodoende dat je geen js nodig hebt ;-)  Dat is per slot van rekening pure html5